### PR TITLE
update wpf sample for code style and readme

### PR DESCRIPTION
**Pull request by xinyzhou**
## Purpose

* Better code style, more clear readme

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[x] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
Build the NoteTaker.sln
```

* Test the code
```
1) Add subscription key to src\MainWindow.xaml.cs
2) Ink some text / draw shapes on the Inking area
```

## What to Check
Verify that the following are valid
* Make sure the recognized text/shape is displayed at the bottom of the app

### CODE REVIEW

File: dotnet/InkRecognition/wpf-app/README.md [hunk 1]
```diff
@@ -1,58 +1,69 @@
 ---
 topic: sample
 languages:
-  - C#
+  - csharp
```
Comment (xinyzhou) [inline dotnet/InkRecognition/wpf-app/README.md]: use c#

File: dotnet/InkRecognition/wpf-app/README.md [hunk 2]
```diff
@@ -5,35 +5,28 @@ languages:
 products:
   - WPF
   - azure
-  - windows
 ---
 
-# Ink Intelligence Cognitive Service C# WPF Sample
+# Ink Recogizer Cognitive Service C# WPF Sample
```
Comment (gowthu2712) [inline dotnet/InkRecognition/wpf-app/README.md]: Recognizer

File: dotnet/InkRecognition/wpf-app/src/NoteTaker.csproj [hunk 3]
```diff
@@ -78,7 +78,6 @@
     <Reference Include="System.Runtime.Serialization" />
     <Reference Include="System.Runtime.WindowsRuntime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>C:\Windows\Microsoft.NET\Framework\v4.0.30319\System.Runtime.WindowsRuntime.dll</HintPath>
```
Comment (gowthu2712) [inline dotnet/InkRecognition/wpf-app/src/NoteTaker.csproj]: Not needed?

File: dotnet/InkRecognition/wpf-app/src/Services/Ink/InkRecognizer.cs [hunk 4]
```diff
@@ -17,7 +17,7 @@ public class InkRecognizer
 
         HttpManager httpManager;
 
-        // Use default DPI setting
+        // Default DPI setting. 96.0 is Windows default DPI
```
Comment (gowthu2712) [inline dotnet/InkRecognition/wpf-app/src/Services/Ink/InkRecognizer.cs]: This is redundant

File: dotnet/InkRecognition/wpf-app/README.md [hunk 5]
```diff
@@ -42,17 +35,35 @@ This project framework provides the following features:
...
+
+1. Clone or download this sample repository `git clone https://github.com/Azure-Samples/cognitive-services-csharp-wpf-ink-recognition.git`
+2. Start Visual Studio and select **File > Open > Project/Solution** and open **src\NoteTaker.sln**
+3. Replace "[YOUR SUBSCRIPTION KEY]" in **src\MainWindows.xaml.cs** with valid subscription key
+4. Select **Build > Build Solution**
+
+### Run the sample
+
+* To debug the sample and then run it, select **Debug > Start Debugging**. To run the sample without debugging, select **Debug > Start Without Debugging**.
+* Write down something one the upper rectangle of the app
```
Comment (gowthu2712) [inline dotnet/InkRecognition/wpf-app/README.md]: on

Comment (gowthu2712) [reply dotnet/InkRecognition/wpf-app/README.md]: "rectangle" might be confusing! Please repharse the instruction. Same applies to L58

File: dotnet/InkRecognition/wpf-app/src/NoteTaker.csproj (hunk 3 above)
Comment (xinyzhou) [reply dotnet/InkRecognition/wpf-app/src/NoteTaker.csproj]: I tried some fresh machine and this line is not needed. VS can find a WindowsRuntime at somewhere under C:\Windows\Microsoft.Net\assembly

File: dotnet/InkRecognition/wpf-app/src/MainWindow.xaml.cs [hunk 6]
```diff
@@ -19,6 +19,7 @@ public partial class MainWindow : Window
         InkRecognizer inkRecognizer;
         DisplayInformation displayInfo;
         private readonly DispatcherTimer dispatcherTimer;
+        // Time to wait before triggering ink recognition operation
```
Comment (gowthu2712) [inline dotnet/InkRecognition/wpf-app/src/MainWindow.xaml.cs]: Can you also add a comment before L29 : 
// Replace the subscriptionKey string value with your valid subscription key.

File: dotnet/InkRecognition/wpf-app/src/NoteTaker.csproj (hunk 3 above)
Comment (gowthu2712) [reply dotnet/InkRecognition/wpf-app/src/NoteTaker.csproj]: Got it!

File: dotnet/InkRecognition/wpf-app/README.md (hunk 5 above)
Comment (xinyzhou) [reply dotnet/InkRecognition/wpf-app/README.md]: Is "region" better?

File: dotnet/InkRecognition/wpf-app/src/Services/Ink/InkRecognizer.cs (hunk 4 above)
Comment (xinyzhou) [reply dotnet/InkRecognition/wpf-app/src/Services/Ink/InkRecognizer.cs]: use the comment in your sample

File: dotnet/InkRecognition/wpf-app/README.md [hunk 7]
```diff
@@ -42,17 +35,36 @@ This project framework provides the following features:
...
+### Build the Sample
+
+1. Clone or download this sample repository `git clone https://github.com/Azure-Samples/cognitive-services-csharp-wpf-ink-recognition.git`
+2. Start Visual Studio and select **File > Open > Project/Solution** and open **src\NoteTaker.sln**
+3. Replace "[YOUR SUBSCRIPTION KEY]" in **src\MainWindows.xaml.cs** with valid subscription key
+4. Select **Build > Build Solution**
+
+### Run the sample
+
+* To debug the sample and then run it, select **Debug > Start Debugging**. To run the sample without debugging, select **Debug > Start Without Debugging**.
```
Comment (olduroja) [inline dotnet/InkRecognition/wpf-app/README.md]: remove "and then run it" its confusing

**Pull request merged**